src/main/java/frc/robot/subsystems/Elevator/ElevatorIO.java

```diff
@@ -27,6 +29,6 @@ public default void updateInputs(ElevatorIOInputs inputs) {}
 
   public void setDutyCycle(double dutyCycle);
 
-  public boolean getBottomSwitch();
+  public double getPosition();
```

Comment:
We shouldn't have getter methods in the io layer because everything should be handled by the inputs from the io layer